### ISSUE TITLE
Reporting of hardware errors

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#1097]
Windows logs hardware errors
![image](https://user-images.githubusercontent.com/56473344/87313166-65250280-c519-11ea-849b-87249301b9d0.png)

I find that sometimes there are crashes that Windows detects but RBM does not. Could this be added to RBM monitoring and reporting?

**Comment by RainbowMiner** [comment]
Yes, that could be useful. Let me have look into the Windows hardware error log files. If possible, I'll try to add some kind of reader.

**Comment by samdave123** [comment]
I quite like how Awesome Miner shows HW errors for cards, maybe they're getting it from the Windows error log somehow? https://cdn.discordapp.com/attachments/446124403051200534/738042279532560404/unknown.png

**Comment by jch9678** [comment]
Along the same lines I would love if Monitoring Central showed all the same info that Awesome Miner does.  I've mentioned that here https://github.com/RainbowMiner/RainbowMiner/issues/297 and sort of here https://github.com/RainbowMiner/RainbowMiner/issues/1010
You've pretty much added the payout request from issue 297.
I would happily use RainbowMiner and pay the fee on my miners that don't need profit switching just for Awesome Miner like reporting and monitoring.
